src/config/pragmas/plugins.js

```diff
@@ -0,0 +1,23 @@
+const { join } = require('path')
+const { existsSync } = require('fs')
+
+module.exports = function configurePlugins ({ arc, inventory }) {
+  if (!arc.plugins || !arc.plugins.length) return {}
```

Comment:
Inventory defaults to `null` if not user-defined (with a handful of exceptions, usually related to Arc system defaults), returning an empty object would not be consistent with the API.